pyfr/plugins/writer.py

```diff
@@ -32,11 +32,9 @@ def __init__(self, intg, cfgsect, suffix=None):
         # If we're not restarting then write out the initial solution
         if not intg.isrestart:
```

Comment:
Double check that behaviour here has not changed (normal run and restart for when start/restart time is/is not a multiple).

Reply:
The sanity checks seem to work and the tavg fields look right. I tried
1. Fresh start with dt-out that is multiple of dt works
2. Fresh start with dt-out that is not multiple of dt fails
3. Restart with dt-out that is multiple of dt works. (if dt is changed you need to also change dt-out)
4. Restart with dt-out that is not multiple of dt fails

Standard stepper works with any dt-out.